pyproject.toml

```diff
@@ -41,6 +41,8 @@ dependencies = [
     # numba technically optional, will still work without it
     "numba",
     "pyarrow",
+    "scipy",
+    "skan",
```

Comment:
I should have this release done today. Before merging, this should be `skan>=0.13.0`.

Reply:
0.13.0 is now out in PyPI and conda-forge